slackchat/models/custom_content_template.py

```diff
@@ -8,19 +9,36 @@ class CustomContentTemplate(models.Model):
     """
 
     name = models.CharField(max_length=255)
-    argument_name = models.SlugField(
-        max_length=255,
-        blank=True, null=True,
-        help_text="Add an argument to the message if search_string \
-        matches against a message's content.")
     search_string = models.CharField(
-        max_length=255,
-        help_text="A regex search string with capture groups.")
-    chat_type = models.ForeignKey(
-        'ChatType', on_delete=models.CASCADE)
+        max_length=255, help_text="A regex search string with capture groups."
+    )
+    chat_type = models.ForeignKey("ChatType", on_delete=models.CASCADE)
     content_template = models.TextField(
+        blank=True,
+        null=True,
         help_text="A Python format string whose args are the capture groups \
-        matched by the search_string."
+        matched by the search_string.",
+    )
+    argument_template = models.CharField(
+        blank=True,
+        null=True,
+        max_length=255,
+        help_text="A comma seprated list of Python format string whose \
+        args are the capture groups matched by the search_string.",
+    )
+    attachment_template = JSONField(
+        blank=True,
+        null=True,
+        help_text="A JSON object with Python formatted strings whose args are \
```

Comment:
How about: `A JSON object added to a message's attachments whose values may be Python formatted strings.  The args given to formatted strings are the capture groups matched by the search string.`